sync_diff_inspector/source/source.go

```diff
@@ -45,8 +45,9 @@ const (
 const UnifiedTimeZone string = "+0:00"
 
 const (
-	ShieldDBName    = "_no__exists__db_"
-	ShieldTableName = "_no__exists__table_"
+	ShieldDBName      = "_no__exists__db_"
+	ShieldTableName   = "_no__exists__table_"
+	GetSyncPointQuery = "SELECT primary_ts, secondary_ts FROM tidb_cdc.syncpoint_v1 ORDER BY primary_ts DESC LIMIT 1"
```

Comment:
shall we check the table exists before we execute the query?

Reply:
I don't think you need to - the error message is pretty understandable when the query fails (see manual test).